tools/filters/sorter.py

```diff
@@ -35,16 +36,23 @@ def main():
         # retrieve options
         input = options.input
         output = options.output
+        header_lines = options.header_lines
         key = [" -k" + k for k in options.key]
 
+        # sed header
+        if header_lines > 0:
+            sed_header = "sed -n '1,%dp' %s > %s" % (header_lines, input, output)
```

Comment:
On the dev branch I get three passing tests with planemo on Mac OS X, against this branch the first two fail and the last two succeed.